### ISSUE TITLE
hotkeys not working

### ISSUE THREAD

**Issue by netfun81** [yazgoo/umberwm#35]
I've tried umberwm in the past and just updated following the new README. My hotkeys would not work.  After awhile i realized it needed a "mod4" parameter to select the right modifier.  I added "mod4" after umberwm in umberwm-start, as shown below and all seems to be working fine now.  

umberwm mod4 &>> "$HOME"/.local/share/umberwm/umberwm.log

**Comment by mfdorst** [comment]
Yup. I actually think it might be simpler to just set it in `main.rs` like all the other keybindings. That would look like replacing:
```rust
let args: Vec<String> = env::args().collect();

let meta = if args.len() > 1 && args[1] == "mod4" {
    MOD_MASK_4
} else {
    MOD_MASK_1
}
```
with
```rust
let meta = MOD_MASK_1;
```
and then to change it you would just change that to `MOD_MASK_4`.

Would that be suitable for you, or do you prefer it how it is?

**Comment by netfun81** [comment]
sure that works since the keybinds are also configured in main.rs
    On Friday, May 21, 2021, 09:29:23 PM MDT, Michael Dorst ***@***.***> wrote:  
 
 


Yup. I actually think it might be simpler to just set it in main.rs like all the other keybindings. That would look like replacing:
let args: Vec<String> = env::args().collect();

let meta = if args.len() > 1 && args[1] == "mod4" {
    MOD_MASK_4
} else {
    MOD_MASK_1
}
with
let meta = MOD_MASK_1;
and then to change it you would just change that to MOD_MASK_4.

Would that be suitable for you, or do you prefer it how it is?
